hack/update-generated-docs.sh

```diff
@@ -25,6 +25,9 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+# This only runs on master
+[[ $(git rev-parse --abbrev-ref HEAD) == "master" ]] || exit
```

Comment:
That seems a little racy, but fine.
There are probably a few ways to really tear this apart and refactor it to make contextually Just Work Right every time, but that's probably more than is needed at this point.